build.rs

```diff
@@ -67,13 +67,34 @@ fn gen_cxxbridge() -> Result<()> {
     let out_path = PathBuf::from(out_path).join("gen");
     let src_out_path = PathBuf::from(&out_path).join("src");
     let header_out_path = PathBuf::from(&out_path).join("include").join("rust");
+    let header_out_path_sapling = header_out_path.join("sapling");
+
+    // Copy the files we need to generate the bindings
+    fs::copy(
+        "depend/zcash/src/rust/src/sapling/spec.rs",
+        "depend/zcash/src/rust/src/spec.rs",
+    )
+    .unwrap();
+    fs::copy(
+        "depend/zcash/src/rust/src/sapling/zip32.rs",
+        "depend/zcash/src/rust/src/zip32.rs",
+    )
+    .unwrap();
+
+    // Replace blake2s with blake2b
+    let spec = fs::read_to_string("depend/zcash/src/rust/src/spec.rs").unwrap();
+    let new_spec = spec.replace("blake2s_simd", "blake2b_simd");
```

Comment:
If we don't replace the import name the compilation will fail.

Reply:
`blake2s` and `blake2b` are different cryptographic algorithm variants, so this change could lead to consensus failures:
https://en.wikipedia.org/wiki/BLAKE_(hash_function)#BLAKE2

Reply:
You are right there. Thank you for the feedback.

The problem here is that only `blake2b` is part of the bridge:

https://github.com/zcash/zcash/blob/v5.7.0/src/Makefile.am#L56

So i am not sure if we should have it as an external import or what to do.